program.html

```diff
@@ -1,47 +1,9 @@
-<h1>Preliminary Program</h1>
-<ul>
-  <li>Introduction</li>
-  <li>ROS 2 Threat Model</li>
-  <li>ROS 2 Demo Presentation</li>
-  <li>Alias Discovery Tool</li>
-  <li>Harden using SROS 2</li>
-  <li>Node sandboxing</li>
-  <li>Conclusion</li>
-</ul>
+<h1>Program</h1>
 
-<!--
-* **Lunch Break \| 13:30 - 14:30**
-  * Break between morning and afternoon tutorials
-* **Introduction \| 14:30 - 14:45 \| Gianluca + Ruffin**
-  * Session overview
-  * Gauging the audience
-  * Logistics and reminders
-  * <a href="/ROSCon19_Security_Workshop/content/slides/Introduction.pdf">Slides</a>
-* **Motivation \| 14:45 - 15:30 \| Bernhard**
-  * Cyber threats in Robotics
-  * Existing attacks on ROS
-  * Available pentesting tools
-  * <a href="/ROSCon19_Security_Workshop/content/slides/Motivation.pdf">Slides</a>
-* **Background \| 15:30 - 16:30 \| Gerardo**
-  * Secure DDS spec from OMG
-  * Feature and performance analysis
-  * Hands on classic Shapes Demo
-  * <a href="/ROSCon19_Security_Workshop/content/slides/Background.pdf">Slides</a>
-* **Coffee Break \| 16:30 - 17:00**
-  * A chance for folks to catch up and mingle
-* **SROS2 Basics \| 17:00 - 18:00 \| Mikael**
-  * Implementation details
-  * Installation setup and runtime
-  * Simple test examples
-  * <a href="/ROSCon19_Security_Workshop/content/slides/SROS2_Basics.pdf">Slides</a>
-* **SROS2 Demos \| 18:00 - 18:30 \| Ruffin**
-  * Hands on demos and best practices
-  * Using Comarmor and Keymint
-  * Debugging and monitoring
-  * <a href="/ROSCon19_Security_Workshop/content/slides/SROS2_Demos.pdf">Slides</a>
-* **Concluding Remarks \| 18:30 - 19:00 \| Ruffin + Mikael**
-  * Future initiatives (e.g. user tooling)
-  * Design challenges (e.g. DDS mapping)
-  * Q&A and Open Discussion
-  * <a href="/ROSCon19_Security_Workshop/content/slides/Concluding_Remarks.pdf">Slides</a>
--->
+* What types of security issues can impact your robot: ROS 2 Threat Model \| 14:00 - 14:30 \| Thomas
+* TurtleBot 3 Demo App Presentation \| 14:30 - 15:00 \| Mikael, Ruffin
+* How to encrypt node-to-node communication using SROS2 part 1 \| 15:00 - 15:15 \| Mikael, Ruffin
```

Comment:
Not sure if the "part 1" is necessary here, we can either add a "part 2" for 15 min after the break or just remove the "part 1" from this line.

For consistency I guess it's better to not have a gap in the program so I'll suggest a change below

Reply:
good catch, thanks!